lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -138,6 +142,40 @@ def create_app_version(s3_object)
         eb.create_application_version(options)
       end
 
+      # Wait until EB environment update finishes
+      def wait_until_deployed
+        success = true
```

Comment:
Somewhat pedantic, but I think we should assume `success = false`, then set it to `true` when the upload did succeed.